From 744bf8387be7b3d2ca1cf65670b388a1271c9be7 Mon Sep 17 00:00:00 2001
From: Juan Gutierrez <juan.gutierrez@nxp.com>
Date: Tue, 15 Nov 2016 18:56:53 -0600
Subject: [PATCH 1310/5242] MXSCM-224 dts: imx: not use mdio device for enet
 phy configuration

commit  29bf42143be5dd3edf944ba8ae315e0e52a8955c from
https://source.codeaurora.org/external/imx/linux-imx.git

Using the mdio and ethphy device that is referenced as a phy_handle
by the fec device is not properly handling the resume from suspend.

Signed-off-by: Juan Gutierrez <juan.gutierrez@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sxscm-evb.dts |   11 ++---------
 1 file changed, 2 insertions(+), 9 deletions(-)

diff --git a/arch/arm/boot/dts/imx6sxscm-evb.dts b/arch/arm/boot/dts/imx6sxscm-evb.dts
index 7d3e227..0b5052c 100644
--- a/arch/arm/boot/dts/imx6sxscm-evb.dts
+++ b/arch/arm/boot/dts/imx6sxscm-evb.dts
@@ -253,16 +253,9 @@
 	pinctrl-0 = <&pinctrl_enet1>;
 	phy-supply = <&reg_enet_3v3>;
 	phy-mode = "rgmii";
-	phy-handle = <&ethphy1>;
+	phy-reset-gpios = <&gpio4 26 0>;
+	fsl,magic-packet;
 	status = "okay";
-
-	mdio {
-		#address-cells = <1>;
-		#size-cells = <0>;
-		ethphy1: ethernet-phy@1 {
-			reg = <3>;
-		};
-	};
 };
 
 &csi1 {
-- 
1.7.9.5

